app/serializers/site_serializer.rb

```diff
@@ -9,6 +9,10 @@ def name
     object.location_name
   end
 
+  def region_code
```

Comment:
Won't `ActiveModel::Serializer` automatically refer to `object.region_code` for us for this attribute?

Reply:
Ah, yes it will. I should have spotted that.

Reply:
Let's take this out, then.